src/components/address-text/address-text.tsx

```diff
@@ -1,9 +1,9 @@
-import React from "react";
-import { useQuery } from "react-query";
-import { CopyToClipboard, validateAddress } from "shared/components";
-import { BoxProps, HStack, Text } from "@chakra-ui/react";
-import { Identity } from "@liftedinit/many-js";
-import { makeShortId } from "shared/helpers";
+import React from "react"
+import { useQuery } from "react-query"
+import { CopyToClipboard, validateAddress } from ".."
+import { BoxProps, HStack, Text } from "@chakra-ui/react"
+import { Identity } from "@liftedinit/many-js"
+import { makeShortId } from "../../helpers"
```

Comment:
Unfortunately, the `baseUrl` field in `tsconfig.json` doesn't work when the library is imported so I'm doing ugly relative links for now. There are other ways that we can revisit.